src/components/NcListItem/NcListItem.vue

```diff
@@ -761,11 +761,14 @@ export default {
 
 .list-item-content__name {
 	min-width: 100px;
-	max-width: 300px;
 	flex: 1 1 10%;
 	font-weight: 500;
 }
 
+.limit-width {
```

Comment:
Some thing like this:
```suggestion
.list-item--one-line {
	.list-item-content__name {
		max-width: 300px;
	}
}
```